modules/uncertainty/R/generate_joint_ensemble_design.R

```diff
@@ -18,6 +19,7 @@
 
 generate_joint_ensemble_design <- function(settings,
                                            ensemble_size,
+                                           gen.samples = FALSE,
```

Comment:
* style suggestion: call the arg generate_samples instead of gen.samples
* My instinct is that the default should be TRUE -- at least in the forward run, this is intended to be the single place samples are generated, so skipping it should need explicit user input
* What should happen when generate_samples is TRUE but samples.Rdata already exists? I'd favor skipping it rather than overwriting.
```suggestion
                                           generate_samples = TRUE,
```

Reply:
Fixed.